Source/WebInspectorUI/UserInterface/Models/Cookie.js

```diff
@@ -25,7 +25,7 @@
 
 WI.Cookie = class Cookie
 {
-    constructor(type, name, value, {header, expires, session, maxAge, path, domain, secure, httpOnly, sameSite} = {})
+    constructor(type, name, value, {header, expires, session, maxAge, path, domain, secure, httpOnly, partitionKey, partitioned, sameSite} = {})
```

Comment:
NIT: please match the ordering of properties from `Page.Cookie`

ditto below (i.e. `partitionKey` and `partitioned` should be after `sameSite`)